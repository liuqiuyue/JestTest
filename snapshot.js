export const getConfig = () => {
  return {
      server: 'https://demo.com',
      port: '8080'
  }
}
