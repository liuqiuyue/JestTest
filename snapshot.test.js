import { getConfig } from './snapshot'
test('getConfig测试', () => {
  expect(getConfig()).toMatchSnapshot()
})

