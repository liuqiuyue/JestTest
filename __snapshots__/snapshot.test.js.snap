// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getConfig测试 1`] = `
Object {
  "port": "8080",
  "server": "https://demo.com",
}
`;
